roles/sync/molecule/default/molecule.yml

```diff
@@ -4,7 +4,7 @@ dependency:
 driver:
   name: docker
 platforms:
-  - name: sync
+  - name: instance
```

Comment:
use a different instance name for each role so it's possible to converge more than one molecule test locally at the same time

Reply:
OK I reverted but you should be able to do that jsut by cd'ing to the other folder...

Reply:
if you do a molecule converge while you already have a molecule converge running for a different role, it tries to use the same docker container already running (I think it use the docker container name as a key) xD